Doc/library/os.path.rst

```diff
@@ -304,12 +304,13 @@ the :mod:`glob` module.)
 
 .. function:: join(path, *paths)
 
-   Join one or more path components intelligently.  The return value is the
-   concatenation of *path* and any members of *\*paths* with exactly one
-   directory separator following each non-empty part except the last, meaning
-   that the result will only end in a separator if the last part is empty.  If
-   a component is an absolute path, all previous components are thrown away
-   and joining continues from the absolute path component.
+   Join one or more path components intelligently. The return value is the
```

Comment:
As far as I can tell, the only change on this line is to change two spaces into one.  That's not an inherently *bad* change, but it does make it rather more difficult to tell what actually changed in the wording of this paragraph.  Please refrain from formatting changes like this when they're mixed in with other changes and are the only change on a line.